README.md

```diff
@@ -1,16 +1,74 @@
-# Title
+# Writing Dynamic Programs
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
++ Write a case statement
++ Recognize the comment character
++ Compare running a Ruby file versus using IRB
++ Run a Ruby program in the Learn In-Browser IDE
++ Write an if...end statement
++ Write an if...else...end Statement
++ Write an if...elsif...else...end Statement
++ Use puts statement to send content to the screen
++ Use puts statement to send content to the screen
++ Add a comment to code
++ Comment-Out code
++ Provide the command to run a file through Ruby
++ Translate a case statement from if...elsif...else...end
++ Define a statement versus an expression
++ Identify the nil value and its purpose
++ Use statement modifiers to change default sequence
 
-## Introduction
+## Outline
 
-## SWBAT 1
+This video should cover writing our first application. Should get input from a user, and conditionally print something to the screen.
```

Comment:
I would also be clear about what "dynamic" from the title means, beginners may not grok what that word means in terms of programming.

Reply:
Good call!

Reply:
```suggestion
This video should cover writing our first application. Should get input from a user, and conditionally print something to the screen. This will be our first "dynamic" application - this just means that our user will see different content depending on what type of information they enter. 
```